src/main/xsl/common/dita-utilities.xsl

```diff
@@ -124,11 +124,26 @@ See the accompanying LICENSE file for applicable license.
         </xsl:choose>
       </xsl:when>
       <xsl:otherwise>
-        <xsl:value-of select="$id"/>
-        <xsl:call-template name="output-message">
-          <xsl:with-param name="id" select="'DOTX052W'"/>
-          <xsl:with-param name="msgparams">%1=<xsl:value-of select="$id"/></xsl:with-param>
-        </xsl:call-template>
+        <xsl:variable name="stringfile" select="$stringFiles[@xml:lang='']/@filename" as="xs:string*"/>
+        <xsl:variable name="str" as="element()*">
```

Comment:
The variable name here is a bit misleading. Maybe rename to `variable` or `var`?

Reply:
Sure - though note that I was just using the same variable name we already use a few lines up, when pulling in the locale-specific variables :-)

Reply:
The old variable names are misleading too :)